docs/application-connector/08-10-rotate-root-ca.md

```diff
@@ -115,7 +115,7 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
 
 5. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate. Run:
   ```
-  kubectl -n istio-system edit secret application-connector-ca-certs
+  kubectl -n istio-system edit secret application-connector-certs-cacert
   ```
 
 6. If you experience any issues with new certificates not being trusted, restart Istio Ingress Gateway Pods. Run:
```

Comment:
From what I understand after these changes, we no longer need to restart the pod after changing the content of the secret. Am I wrong on this?